README.md

```diff
@@ -385,6 +385,9 @@ Tests:
 Checks that the application can read configuration from a ConfigMap and a Secret.
 The ConfigMap/Secret is exposed by mounting it into the container file system or the Kubernetes API server.
 
+### `docker-build`
+Checks that an application can generate a Docker image based in some configuration parameters.
```

Comment:
I presume that "in" is a typo, and it should be "on" instead.

Reply:
these files are generated by `mvn io.quarkus.platform:quarkus-maven-plugin:2.6.1.Final:create` command by default. It's true that we are not going to use it, but I prefer to leave it as it is.